string topic
string id
---
string result
